cpp/daal/src/algorithms/linear_regression/oneapi/linear_regression_train_dense_normeq_helper_oneapi_impl.i

```diff
@@ -71,10 +75,13 @@ services::Status KernelHelperOneAPI<algorithmFPType>::copyBetaToResult(const ser
 
```

Comment:
factory.build - check status
factory.getKernel - check status

Reply:
It's just initial improvements)